lnrpc/signrpc/signer_server.go

```diff
@@ -506,38 +490,77 @@ func (s *Server) VerifyMessage(ctx context.Context,
 
 // DeriveSharedKey returns a shared secret key by performing Diffie-Hellman key
 // derivation between the ephemeral public key in the request and the node's
-// key specified in the key_loc parameter (or the node's identity private key
-// if no key locator is specified):
-//     P_shared = privKeyNode * ephemeralPubkey
+// key specified in the key_desc parameter. Then key_family of the key locator
```

Comment:
This should match the comment in the `.proto` file.
I would replace the last sentence with:

> Either a key locator or a raw public key is expected in the key_desc, if neither is supplied, defaults to the node's identity private key. The old key_loc parameter in the request shouldn't be used anymore.

Reply:
fixed